apmpackage/apm/0.3.0/manifest.yml

```diff
@@ -47,6 +47,12 @@ policy_templates:
             required: true
             show_user: true
             default: localhost:8200
+          - name: url
+            type: text
+            title: URL
+            required: true
+            show_user: true
+            default: localhost:8200
```

Comment:
```suggestion
            default: http://localhost:8200
```

?

Reply:
I remember someone saying that it should default to the same value as `host`, but I'm happy to add the `http://`. Not sure what the "?" is asking :)

Reply:
@stuartnelson3 I mentioned to default to the `host` value, but am +1 on adding `http://`

Reply:
that was vague way of saying I'm not 100% sure, but I think it should be a proper URL